src/makers/heltec/hotspots.ts

```diff
@@ -0,0 +1,64 @@
+import HeltecIndoorIcon from './Heltec_Indoor.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const HELTEC_INDOOR = {
+  name: 'Heltec Indoor Hotspot',
+  icon: HeltecIndoorIcon,
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Example Maker to identify issues with your Hotspot in a secure way.</white></b>\n\nHeltec will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@heltec.cn</b></purple> from the email used to purchase the Hotspot.',
```

Comment:
You'll want to update "Example Maker" to your name here.

Reply:
Thank you very much for your hint, I have fixed this problem. Thank you in advance and please review again. 😄